app/api/courses.py

```diff
@@ -29,7 +25,28 @@ def search_course(
     availability: Optional[bool] = None,
 ):
     """
-    Search Courses using filters
+    <p>
+    This is a public, read-only API that allows Aged Care Provider (employer) to:\n
+    the employer is able to:
+    <ul>
+      <li>search for courses meeting various criteria, so they can organise training for staff.</li>
+      <li>see what industry standards the training provider supports</li>
+    </ul>
+    \n
+    It is part of the interface standard because it allows employers to\n
+    federate search results from multiple training providers.\n
```

Comment:
I think you can remove `\n` from these two lines. It makes it difficult to read.

Reply:
I put it there because somehow list was making the font size of following text smaller, it helped to prevent that.

Reply:
Aah, okay. It's weird.